custom_components/aula/manifest.json

```diff
@@ -15,5 +15,5 @@
         "beautifulsoup4",
         "lxml"
     ],
-    "version": "0.1.38"
+    "version": "--path"
```

Comment:
What does this mean exactly? The version in manifest.json is auto incremented by a pipeline.

Reply:
Ooops, that was probably me being a newbie at setting up python build environments. I'll fix this up.